test/Microsoft.Extensions.Configuration.UserSecrets.Test/ConfigurationExtensionTest.cs

```diff
@@ -43,6 +45,35 @@ private void SetSecret(string id, string key, string value)
         }
 
         [Fact]
+        public void AddUserSecrets_FindsAssemblyAttribute()
+        {
+            var configKey = nameof(AddUserSecrets_FindsAssemblyAttribute);
+            SetSecret("Microsoft.Extensions.Configuration.UserSecrets.Test", configKey, "true");
```

Comment:
Not sure I understand this test. What is "true" and "false" here? Is that the "user secrets id"? I.e. the "application name id" that I would use? If so, can we put a clearer dummy value here, e.g. "MyUserSecretsIdValue" or something?